rsv-metrics/libexec/probes/cacert-verify-probe

```diff
@@ -33,6 +33,13 @@ def md5sum(fname):
   f.close()
   return checksum.hexdigest().lower()
 
+def md5sum(fname):
+  """Return the md5 checksum of the file."""
+  return hashfile(hashlib.md5, fname)
+
+def sha256sum(fname):
+  """Return the md5 checksum of the file."""
```

Comment:
this comment should be "Returns the sha256 checksum..."

Reply:
Woops!

Reply:
"Return" not "Returns" -- Python docstrings should be in the imperative tense. Otherwise, Zalak is correct.